lib/ey-core/cli/console.rb

```diff
@@ -0,0 +1,14 @@
+class Ey::Core::Cli::Console < Ey::Core::Cli::Subcommand
+  title "console"
+  summary "Start an interactive console"
+  option :execute_command, short: "c", long: "command", description: "Command to execute", argument: "command"
```

Comment:
I had `-e` in my implementation because of `ruby -e` is `e` need for something else or is there another unixy thing that make `-c` make more sense?